sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/extensions/TestHeaderExtension.java

```diff
@@ -19,14 +19,17 @@ public void visitSpec(SpecInfo specInfo) {
 
     private static class TestHeaderIterationInterceptor implements IMethodInterceptor {
         @Override
-        public void intercept(IMethodInvocation invocation) {
+        public void intercept(IMethodInvocation invocation) throws Throwable {
             // Print out the test name to create breadcrumbs in our test logging in case anything hangs.
             String testName = TestNameProvider.getTestName(invocation.getIteration());
-            System.out.printf("========================= %s =========================%n", testName);
+            System.out.printf("%s is starting", testName);
+            long startTimestamp = System.currentTimeMillis();
+
             try {
                 invocation.proceed();
-            } catch (Throwable throwable) {
-                throwable.printStackTrace();
+            } finally {
```

Comment:
Do we want to keep the `catch` and log the testing error?

Reply:
I think we might have disturbed the test framework by intercepting this in such a way.

If it's exception then test framework should dump it, if it's Error then JVM should dump it into output when it bubbles.